public/index.html

```diff
@@ -8,11 +8,6 @@
     <meta name="description" content="Genny App" />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <link
-      rel="stylesheet"
-      type="text/css"
-      href="//fonts.googleapis.com/css?family=Homemade+Apple"
```

Comment:
@sjlevens Please let me know if moving this font to `src/config/fonts/index.js` broke anything in the signature component.